circuit_knitting/cutting/qpd/instructions/qpd_gate.py

```diff
@@ -101,11 +99,6 @@ def __eq__(self, other):
             and self.label == other.label
         )
 
-    @abstractmethod
-    def _define(self) -> None:
-        """Generate a decomposed gate."""
-        raise NotImplementedError  # pragma: no cover
-
```

Comment:
Removing these lines wasn't essential, but seems more in line with the idea of relying on the base class's `Instruction._define` [to do nothing rather](https://github.com/Qiskit/qiskit/blob/e181eabc04252d332448f49399de9b72ab86b375/qiskit/circuit/instruction.py#L269-L271) than raise an error if a definition isn't explicitly provided.

Reply:
This makes sense to me, thanks.